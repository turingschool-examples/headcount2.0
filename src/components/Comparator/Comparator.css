.Comparator {
  height: 350px;
  padding: 1rem;
  margin-bottom: 10px;
  grid-row: 2 / 3;
  grid-column: 1 / 2;
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: flex-start;
  background-color: #DCF1F4;
  color: #006989;
  text-align: center;
  font-family: 'Roboto', sans-serif;
  font-weight: 400;
  letter-spacing: 0.5px;
}

.Comparator h2 {
  font-family: 'Roboto', sans-serif;
  font-weight: 900;
  margin-bottom: 0;
}

p {
  font-size: 1.15rem;
}