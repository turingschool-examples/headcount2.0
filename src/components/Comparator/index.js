import React from 'react'
import PropTypes from 'prop-types'
import Card from '../Card'
import './Comparator.css'

const Comparator = () => {
  return(
    <section className="Comparator">
    </section>
  )
}

export default Comparator;