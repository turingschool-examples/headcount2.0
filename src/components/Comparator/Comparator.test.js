import React from 'react'
import { shallow } from 'enzyme'
import Comparator from './index'

describe('Comparator', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<Comparator />)
  })

  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  })
})