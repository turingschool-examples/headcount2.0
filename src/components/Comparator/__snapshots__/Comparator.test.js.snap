// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Comparator should match the snapshot 1`] = `
<section
  className="Comparator"
/>
`;
